### PR TITLE
Action menu on list items (again)

### PR DESCRIPTION
**Pull request by jensgustafsson**
Connected to: Lundalogik/crm-feature#989

## Review:
- [x] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [x] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [x] Chrome
- [ ] Firefox

macOS:
- [x] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/components/menu/menu.tsx [hunk 1]
```diff
@@ -124,28 +99,33 @@ export class Menu {
...
     private onTriggerClick = () => {
+        if (this.disabled) {
+            return;
+        }
         this.open = !this.open;
     };
+
+    private onListChange = (event) => {
+        this.items = this.items.map((item: ListItem) => {
+            if (item.value === event.detail.value) {
```
Comment (jensgustafsson) [inline src/components/menu/menu.tsx:122]: I think value has to be a primitive object in order for this to work

Comment (jgroth) [reply src/components/menu/menu.tsx:122]: We should just have to compare the items I guess :thinking: 
```suggestion
            if (item === event.detail) {
```

Comment (jensgustafsson) [reply src/components/menu/menu.tsx]: Then I could probably remove most of my changes in the web client :)

Comment (jgroth) [reply src/components/menu/menu.tsx]: :smile:

File: src/components/portal/portal.tsx [hunk 2]
```diff
@@ -101,6 +102,7 @@ export class Portal {
         const content = slot.assignedElements();
 
         this.container = document.createElement('div');
+        this.container.style.opacity = '0';
```
Comment (jgroth) [inline src/components/portal/portal.tsx:105]: Super minor, but since we have a `showContainer`, we could have a `hideContainer` as well :smile:

Comment (jensgustafsson) [reply src/components/portal/portal.tsx:105]: Yea thats true! I will change that :)

**Pull request merged**